pkg/runner/job_runner.go

```diff
@@ -256,10 +271,54 @@ func (jr *JobRunner) Run(j *job.Job) ([][]*job.Report, []*job.Report, error) {
 	return allRunsReports, finalReports, nil
 }
 
+// GetCurrentRun returns the run which is currently being executed
+func (jr *JobRunner) GetCurrentRun(jobID types.JobID) (*types.RunID, error) {
```

Comment:
Just out of curiosity:

Why `*types.RunID`? There could never be a case so the value will be `nil` while error is not `nil`. So the error could be used as the detector of validity.